suite-common/redux-utils/src/hooks/useSelectorDeepComparison.ts

```diff
@@ -0,0 +1,11 @@
+import { useSelector } from 'react-redux';
+
+import isEqualWith from 'lodash/isEqualWith';
+
+export const useSelectorDeepComparison: typeof useSelector = (selector, equalityFn) => {
```

Comment:
Would be great to add some explanation comment when it's useful to consider this hook. I guess it's when there is a performance issue and measurement shows that this is faster in that case?

Reply:
Sometimes especially selectors derived from accounts returns new object/array everytime when account is modified even if it doesn't affect output so it will cause component rerender for no reason and this kind of last resort solution. I will try to think about some general note for this selector.

But I am really unhappy with state of selectors now, they are sometimes slow and what's worse that they are sometimes unreliable. I will need to give this proper investigation.